stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/list/ChannelListView.java

```diff
@@ -0,0 +1,190 @@
+package io.getstream.chat.android.ui.channel.list;
```

Comment:
Ports `ChannelListView` from old module to new so that VH factory could be modified. Also want to convert this to Kotlin.

Reply:
Sure, all new code "should be written" in Kotlin